asynchronous-authorization/langchain-fastapi-py/backend/pyproject.toml

```
@@ -6,7 +6,7 @@ readme = "README.md"
 requires-python = ">=3.13"
 dependencies = [
     "auth0-ai-langchain>=1.0.0b4",
```

we should bump this one too: 
```suggestion
    "auth0-ai-langchain>=1.0.0b5",
```
ref. https://pypi.org/project/auth0-ai-langchain/1.0.0b5/